data/transactions/verify/txn.go

```diff
@@ -36,6 +36,12 @@ import (
 var logicGoodTotal = metrics.MakeCounter(metrics.MetricName{Name: "algod_ledger_logic_ok", Description: "Total transaction scripts executed and accepted"})
 var logicRejTotal = metrics.MakeCounter(metrics.MetricName{Name: "algod_ledger_logic_rej", Description: "Total transaction scripts executed and rejected"})
 var logicErrTotal = metrics.MakeCounter(metrics.MetricName{Name: "algod_ledger_logic_err", Description: "Total transaction scripts executed and errored"})
+var msigLessOrEqual4 = metrics.MakeCounter(metrics.MetricName{Name: "algod_verify_msig_4", Description: "Total transactions with 1-4 msigs"})
+var msigLessOrEqual16 = metrics.MakeCounter(metrics.MetricName{Name: "algod_verify_msig_4_16", Description: "Total transactions with 4-16 msigs"})
+var msigMore16 = metrics.MakeCounter(metrics.MetricName{Name: "algod_verify_msig_16", Description: "Total transactions with 16+ msigs"})
+var msigLsigLessOrEqual4 = metrics.MakeCounter(metrics.MetricName{Name: "algod_verify_msig_lsig_4", Description: "Total transaction scripts with 1-4 msigs"})
+var msigLsigLessOrEqual16 = metrics.MakeCounter(metrics.MetricName{Name: "algod_verify_msig_lsig_4_16", Description: "Total transaction scripts with 4-16 msigs"})
+var msigLsigMore16 = metrics.MakeCounter(metrics.MetricName{Name: "algod_verify_msig_lsig_16", Description: "Total transaction scripts with 16+ msigs"})
```

Comment:
nit: (personal preference...)
```suggestion
var msigLsig17orMore = metrics.MakeCounter(metrics.MetricName{Name: "algod_verify_msig_lsig_17", Description: "Total transaction scripts with 17 or more msigs"})
```